Makefile

```diff
@@ -1,36 +1,86 @@
-IMAGE_TAG := $(shell git rev-parse --short=7 HEAD)
+#
+# ClusterIQ Console Makefile
+################################################################################
+
+# Global Vars
+SHORT_COMMIT_HASH := $(shell git rev-parse --short=7 HEAD)
+
+# Binary vars
 CONTAINER_ENGINE ?= $(shell which podman >/dev/null 2>&1 && echo podman || echo docker)
 K8S_CLI ?= $(shell which oc >/dev/null 2>&1 && echo oc || echo kubectl)
+
+# Container image registy vars
 REGISTRY ?= quay.io
 PROJECT_NAME ?= cluster-iq
 REGISTRY_REPO ?= ecosystem-appeng
+COMPOSE_NETWORK ?= compose_cluster_iq
+
+# Project directories
+DEPLOYMENTS_DIR ?= ./deployments
+
+# Images
 CONSOLE_IMG_NAME ?= $(PROJECT_NAME)-console
 CONSOLE_IMAGE ?= $(REGISTRY)/$(REGISTRY_REPO)/$(CONSOLE_IMG_NAME)
 
-install: ## Install project dependencies
+
+# Standard targets
+all: ## Installs and run the Console on Local
+all: install build-local start-dev
+
+# Local working targets
+local-clean: ## Cleans project built (./dist)
+	@echo "### [Cleanning local building] ###"
+	@npm run clean
+
+local-install: ## Installs project dependencies in local
 	@echo "### [Installing dependencies] ###"
 	@npm install
 
-build-local: ## Build the project locally
+local-build: ## Builds the project locally
 	@echo "### [Building project] ###"
 	@npm run build
 
-build-container: ## Build the project's container image
-	@echo "### [Building project's container image] ###"
+local-start-dev: export VITE_CIQ_API_URL = http://localhost:8081
+local-start-dev: ## Starts the project in development mode
+	@echo "### [Starting project DEV MODE] ###"
+	@echo "### [API-URL: $$VITE_CIQ_API_URL] ###"
+	@npm run start
+
+
+# Container based working targets
+clean: ## Removes the container images for the API and the Scanner
+	@echo "### [Cleanning Container images] ###"
+	@$(CONTAINER_ENGINE) images | grep $(PROJECT_NAME) | awk '{print $$3}' | xargs $(CONTAINER_ENGINE) rmi -f
```

Comment:
I think the filter should be grep "$(REGISTRY_REPO)/$(PROJECT_NAME)"
Otherwise, it will delete the images we use for testing (for example my local images quay.io/vbelouso/cluster-iq-console)